src/Constructorio_NET.Tests/models/Catalog/PatchSearchabilitiesRequestTest.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using Constructorio_NET.Models;
+using Constructorio_NET.Utils;
+using NUnit.Framework;
+
+namespace Constructorio_NET.Tests
+{
+    [TestFixture]
+    public class PatchSearchabilitiesRequestTest
+    {
+        private readonly string Section = "Search Suggestions";
+        private readonly List<Searchability> Searchabilities = new List<Searchability>();
+
+        [OneTimeSetUp]
+        public void Setup()
+        {
+            this.Searchabilities.Add(new Searchability() { Name = "testSearchability", ExactSearchable = true });
+        }
+
+        [Test]
+        public void GetRequestParameters()
```

Comment:
Should we flesh out this test a bit more to test/check all fields?

Reply:
Section is the only field that is used in getRequestParameters. Do you have anything in mind?

Reply:
Ooo, got it. Nope, looks good to me!